From: SeongJae Park <sj@kernel.org>
Date: Mon, 2 Dec 2024 10:56:22 -0800
Subject: [PATCH] mm/damon/sysfs-schemes: fixup 'parametr not described in'
 warning of damos_sysfs_populate_region_dir()

damos_sysfs_populate_region_dir() kerneldoc comment is not describing
the function parameters, and it causes compiler warning.  Fix it.

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202412020702.S7FyZSzV-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/sysfs-schemes.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/mm/damon/sysfs-schemes.c b/mm/damon/sysfs-schemes.c
index 24ba20e5743b..d289d0736e15 100644
--- a/mm/damon/sysfs-schemes.c
+++ b/mm/damon/sysfs-schemes.c
@@ -2088,6 +2088,12 @@ void damon_sysfs_schemes_update_stats(
 
 /**
  * damos_sysfs_populate_region_dir() - Populate a schemes tried region dir.
+ * @sysfs_schemes:	Schemes directory to populate regions directory.
+ * @ctx:		Corresponding DAMON context.
+ * @t:			DAMON target of the @r.
+ * @r:			DAMON region to populate the directory for.
+ * @s:			Corresponding scheme.
+ * @total_bytes_only:	Whether the request is for bytes update only.
  *
  * Called from DAMOS walk callback while holding damon_sysfs_lock.
  */
-- 
2.39.5

